Monika After Story/game/screens.rpy

```diff
@@ -33,7 +33,11 @@ init -1 python:
         "and offer some comments based on what you're doing."
     )
 
-
+    _TXT_FINISHED_UPDATING = (
+        "The updates have been installed. Please reopen Monika After Story.\n\n"
+        "See the patch notes and get spritepacks {a=https://github.com/Monika-After-Story/MonikaModDev/releases/latest}{i}{u}here{/u}{/i}{/a}.\n"
```

Comment:
this should be linked to the website (http://monikaafterstory.com/releases.html)

Reply:
thoughts of instead changing the link each release to the actual patch notes for that release instead?

Reply:
using direct download from the website has reduced the amount of users installing source versions.

Reply:
we aren't asking them to download the release, the only people that will ever see this are people that have already updated automatically.

Reply:
spritepacks should be downloaded from the website as well. Anything to limit the chance for users to dl source versions. I'm fine with two links, one for patch notes and one for spritepacks.

Reply:
so a link to the actual zip file then for the sprite pack? that's the only link for the sprite pack directly

Reply:
no, link to the releases page for spritepacks. "Latest Spritepacks" is listed there front and center.